apps/src/StudioApp.js

```diff
@@ -15,7 +15,7 @@ import * as aceMode from './acemode/mode-javascript_codeorg';
 import * as assetPrefix from './assetManagement/assetPrefix';
 import * as assets from './code-studio/assets';
 import * as blockUtils from './block_utils';
-import * as codegen from './lib/tools/jsinterpreter/codegen';
+var codegen = require('./lib/tools/jsinterpreter/codegen');
```

Comment:
Was it necessary to switch to a require statement here?

Reply:
Yes.  We don't understand the root issue but had issues like this a couple times.  @davidsbailey helped look at it.

Reply:
@islemaster we didn't get to the bottom of it, but the symptom we saw was that the methods we expected to find on `codegen` were instead located on `codegen.default`. I suspect there is something going on related to an ordering dependency where the first time something gets imported (or required) has unintended consequences later when it gets required (or imported). One speculative fix here would be to move all inclusions of `codegen` and `project` to only use `import`, never `require`.